packages/gestalt/src/Typeahead.js

```diff
@@ -0,0 +1,351 @@
+// @flow strict
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import InputField from './TypeaheadInputField.js';
+import Option from './TypeaheadOption.js';
+import Box from './Box.js';
+import Text from './Text.js';
+import Flyout from './Flyout.js';
+import Layer from './Layer.js';
+
+type TextColors =
```

Comment:
@simkessy can we remove all these props and just keep the ones specified in the docs? We don't want

* noResultTextColor
* hoverColor
* textColor
* backgroundColor